docs/sphinx_source/guide/QLIPP.md

```diff
@@ -0,0 +1,58 @@
+# waveorder
+
+`waveorder` is a collection of computational imaging methods. It currently provides QLIPP (quantitative label-free imaging with phase and polarization), phase from defocus, and fluorescence deconvolution.
+
+These are the kinds of data you can acquire with `waveorder` and QLIPP:
+
+https://user-images.githubusercontent.com/9554101/271128301-cc71da57-df6f-401b-a955-796750a96d88.mov
+
+https://user-images.githubusercontent.com/9554101/271128510-aa2180af-607f-4c0c-912c-c18dc4f29432.mp4
+
```

Comment:
These should be embedded so a user doesn't have to download them to view a .mov / .mp4 / .gif etc...